easybuild/easyconfigs/q/Qt/Qt-4.8.6-intel-2015a.eb

```diff
@@ -1,19 +1,32 @@
+#easyblock = 'CMakeMake'
```

Comment:
Ok, but why?  That seems somewhat inconsistent with having to add easyblock = 'ConfigureMake'.